### PR TITLE
Generate new library version

### PR DESCRIPTION
**Pull request by brettmortensen**
First cut at generating a new version of the wrapper library using
code-gen tools. Any feedback is appreciated 👍 

@sambev

### CODE REVIEW

**Comment by minond** [comment]
Similar thoughts as to what I mentioned over on the node PR, https://github.com/mxenabled/mx-atrium-node/pull/19#issuecomment-447045810

It'd be nice to be able to just do:

```php
<?php

$config = new atrium\Configuration();
$config->setApiKey("MX-API-Key", "...");
$config->setApiKey("MX-Client-ID", "...");

$client = new atrium\Client($config);

$client->accounts->listAccountTransactions(...);
$client->members->createMember(...);
```

A wrapper class like this one may should help with this (not tested so...):

```php
<?php

namespace atrium;

use atrium\Api\AccountsApi;
use atrium\Api\MembersApi;
// ...

use GuzzleHttp\ClientInterface;

class Client
{
  public $accounts;
  public $members;

  public __construct(Configuration $config, ClientInterface $httpClient = null)
  {
    $this->accounts = new AccountsApi($httpClient, $config);
    $this->members = new MembersApi($httpClient, $config);
    // ...
  }
}
```

@brettmortensen,  @sambev thoughts?

**Comment by brettmortensen** [comment]
@minond Same as you Node library suggestion, I think this is the way it should be. No need to create individual api clients for each resource. I just added a commit with this wrapper class.

File: MXClient-php/docs/Api/AccountsApi.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+This endpoint allows you to see every transaction that belongs to a specific account. The default from_date is 90 days prior to the request, and the default to_date is 5 days from the time of the request.<br> The from_date and to_date parameters can optionally be appended to the request.
+
+### Example
+```php
+<?php
+require_once(__DIR__ . '/vendor/autoload.php');
+
+// Configure API Key authorization
+$config = atrium\Configuration::getDefaultConfiguration()->setHeader('MX-API-Key', 'YOUR_API_KEY');
```
Comment (sambev) [inline MXClient-php/docs/Api/AccountsApi.md]: Can we configure multiple headers at once? That would be nice to have.

Comment (minond) [reply MXClient-php/docs/Api/AccountsApi.md]: I believe that method returns `$this` so we can do `$x->setHeader('MX-API-Key', 'YOUR_API_KEY')->setHeader('something_else', 'blah');`

Comment (brettmortensen) [reply MXClient-php/docs/Api/AccountsApi.md]: Good idea. We did something similar in the Go library. I've just updated it so we pass the api key and client id in when creating an AtriumClient.

```
$client = new atrium\Api\AtriumClient(
    "YOUR_API_KEY",
    "YOUR_CLIENT_ID",
    new GuzzleHttp\Client()
);
```

**Pull request merged**